src/components/messenger/feed/components/post-input/index.tsx

```diff
@@ -15,12 +17,14 @@ import './styles.scss';
 import Menu from '../../../../message-input/menu/menu';
 import { MediaType } from '../../../../../store/messages';
 import { Media, addImagePreview, dropzoneToMedia, windowClipboard } from '../../../../message-input/utils';
+import { EmojiPicker } from '../../../../message-input/emoji-picker/emoji-picker';
```

Comment:
When we have updated posts implementation i.e. irys, we can work on refactoring the locations of these imports